Providers/Modules/Plugins/ChangeTracking/plugin/changetracking_lib.rb

```diff
@@ -286,8 +286,24 @@ def self.setHash(prev_hash, last_upload_time, file_path)
                 end
     end
 
+    def self.setInventoryTimestamp(timestamp, file_path)
+        File.open(file_path, "w+", 0644) do |f| 
+             f.puts "#{timestamp}"
+        end
+    end
+
+    def self.getInventoryTimestamp(file_path)
+        time = Time.now - (24 * 60 * 60)
```

Comment:
same comment about adding miliseconds or seconds to the time parameters and get api.